01-main/packages/glow

```diff
@@ -0,0 +1,11 @@
+DEFVER=1
+ARCHS_SUPPORTED="i386 amd64 arm64 armhf"
+get_github_releases "charmbracelet/glow" "latest"
+if [ "${ACTION}" != prettylist ]; then
+    URL="$(grep "browser_download_url.*${HOST_ARCH}\.deb\"" "${CACHE_FILE}" | head -n1 | cut -d '"' -f4)"
+    VERSION_PUBLISHED="$(echo "${URL}" | cut -d '_' -f2)"
+fi
+EULA="MIT License"
```

Comment:
That was a surprise! Not sure if `deborah` will be broken by that. @ymauray may need to have a heads-up if it stays.

Reply:
I've removed the EULA for now.